textSearch.java

```diff
@@ -0,0 +1,41 @@
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class textSearch{
+	
+	public static void searchText(String searchTerm, String path) throws IOException{
+		printResult(searchTerm, fileToText(path));
+	}
+	
+	private static List<String> fileToText(String path) throws IOException{
+		java.util.List<String> lines = Files.readAllLines(Paths.get(path));
+		return lines;
+	}
+	
+	private static void printResult(String searchTerm, List<String> lines){
+		for(String s: lines){
+			if(s.contains(searchTerm)){
+				System.out.println(s);
+			}
+		}
+	}
+	
+	public static void main(String[] args){	    

```

Comment:
Format: search searchterm path? Maybe for other function implementation, but ofc not needed to have 3 arguments at this point